client/src/components/lists/MembershipOption/option.tsx

```diff
@@ -44,28 +54,30 @@ function Option({ classes, option, isLast }: Props) {
   };
 
   return (
-    <div
-      onMouseEnter={() => setShowDescription(true)}
-      onMouseLeave={() => setShowDescription(false)}
-      className={classnames(classes.container, {
-        [classes.separator]: !isLast,
-      })}
-    >
-      <Typography
-        variant="h4"
-        align="center"
-        className={classnames(classes.text, classes.title)}
+    <Slide direction="up" in={showOptions} timeout={250}>
+      <div
+        onMouseEnter={() => setShowDescription(true)}
+        onMouseLeave={() => setShowDescription(false)}
+        className={classnames(classes.container, {
+          [classes.separator]: !isLast,
+        })}
       >
-        {option.title}
-      </Typography>
-      {showDescription && (
-        <Typography variant="body2" align="center" className={classes.text}>
-          {option.description}
+        <Typography
+          variant="h4"
+          align="center"
+          className={classnames(classes.text, classes.title)}
+        >
+          {option.title}
         </Typography>
-      )}
-      <BWButton onClick={openAuthPanel}>{option.cta}</BWButton>
-    </div>
+        {(useMediaQuery(theme.breakpoints.down('sm')) || showDescription) && (
```

Comment:
I've never had a good reason to use hooks inside the JSX but I kinda like this 👍 👍

Reply:
Yea. If we start using the same breakpoint in the same file, then we should extract to a variable.
But this is only being used here I think...